test/tests/kernels/ad_scalar_kernel_constraint/diffusion_bipass_scalar.i

```diff
@@ -0,0 +1,157 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  xmin = -1
+  xmax = 1
+  ymin = -1
+  ymax = 1
+  nx = 2
+  ny = 2
+  elem_type = QUAD9
+[]
+
+[Functions]
+  [./exact_fn]
```

Comment:
It would be nice if you could remove the `./` and `../` syntax from tests. I know there are many tests that still use this old syntax but we are trying to slowly eliminate it from the repo